tests/system-containers/main.yml

```diff
@@ -353,12 +369,16 @@
 
     - name: Fail when rootfs dir is missing from main container
       fail:
-        msg: "/var/lib/containers/atomic/{{ g_hw_name }}/rootfs does not exist"
+        msg: |
+          Expected: /var/lib/containers/atomic/{{ g_hw_name }}/rootfs exists is True
+          Actual: /var/lib/containers/atomic/{{{ g_hw_name }}/rootfs exists is {{ hw.stat.exists }}
```

Comment:
extra opening curly brace

Reply:
I'm surprised Ansible didn't choke on this when I tested the test